modules/library/data/src/test/java/org/geotools/data/memory/MemoryDataStoreTest.java

```diff
@@ -161,6 +161,10 @@ public void testMemoryDataStore() throws Exception {
     public void testMemoryDataStoreFeatureCollection() throws IOException {
         DataStore store = new MemoryDataStore(DataUtilities.collection(roadFeatures));
```

Comment:
Looking at it, it did not seem a valid test as there is no call to addFeatures. I see the constructor calls in internally, but this could change in the future. Maybe add an explicit test too?

Reply:
I can add an explicit one for addFeatures as well.